payments/nas/sources/sources.go

```diff
@@ -20,17 +20,20 @@ type (
 	}
 
 	RequestIdSource struct {
-		Type          payments.SourceType `json:"type,omitempty"`
-		Id            string              `json:"id,omitempty"`
-		Cvv           string              `json:"cvv,omitempty"`
-		PaymentMethod string              `json:"payment_method,omitempty"`
+		Type              payments.SourceType `json:"type,omitempty"`
+		Id                string              `json:"id,omitempty"`
+		Cvv               string              `json:"cvv,omitempty"`
+		PaymentMethod     string              `json:"payment_method,omitempty"`
+		Stored            *bool               `json:"stored,omitempty"`
```

Comment:
Please also add `Stored` and `StoreForFutureUse` for RequestIdSource ABC

Reply:
Are we sure that these fields need to be nullable? Being bool fields and using the omitempty tag, means that if the field is not set to true, we don't parse it when serializing to json @Armando-Ibarra

Reply:
In previous version we don't find this properties in the API reference, but should we add them anyway?